src/components/LandingSection.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import {
+  Grid,
+  Row,
+  Column,
+  TextInput,
+  Button,
+  ToastNotification,
+} from "carbon-components-react";
+
+const LandingSection = () => {
+  const [error, setError] = useState("");
+  const [id, setId] = useState("");
+  const [showToaster, setShowToaster] = useState(false);
+
+  const handleBtnClose = () => {
+    setShowToaster(false);
+  };
+
+  const checkInput = (val) => {
+    setError("");
+    setId(val);
+  };
+
+  function isAlphanumeric(str) {
+    return /^[a-zA-Z0-9]+$/.test(str);
+  }
+
+  const submitID = (e) => {
+    e.preventDefault();
+    if (!isAlphanumeric(id)) {
+      setError("Only alphanumeric input is allowed");
+    } else {
+      setError("");
+      setId("");
+      setShowToaster(true);
+    }
+  };
+
+  return (
+    <Grid data-testid="landing-section-container" className="hero__container">
+      <Row className="hero__content">
+        <Column className="hero__text">
+          <div className="hero__section">
+            <h1>
+              Welcome to <span>Patient Portal</span> App
+            </h1>
+            <p>Patient-Centric Care Made Accessible</p>
+            <p>
+              Download and Track all your <span>Medical Records</span> with ease
+            </p>
+            <div className="hero__form">
+              <p>
+                Enter your hospital ID to <span>get started</span>
```

Comment:
"Enter your Patient ID" (not hospital ID).

Reply:
Okay, will change